src/openrct2/config/Config.cpp

```diff
@@ -206,9 +205,9 @@ namespace Config
             model->last_save_landscape_directory = reader->GetCString("last_landscape_directory", nullptr);
             model->last_save_scenario_directory = reader->GetCString("last_scenario_directory", nullptr);
             model->last_save_track_directory = reader->GetCString("last_track_directory", nullptr);
+            model->scroll_wheel_behaviour = reader->GetInt32("scroll_wheel_behaviour", 1);
```

Comment:
Zooming to cursor should still be the default. You've assigned this `2`, not `1`.